code/WorkInProgress/AzrunStuff.dm

```diff
@@ -922,7 +922,7 @@ ADMIN_INTERACT_PROCS(/turf/unsimulated/floor, proc/sunset, proc/sunrise, proc/se
 	check_match(obj/item/craftingitem)
 		if(..() && length(reagents_req))
 			var/obj/item/reagent_containers/RC = craftingitem
-			if(RC.is_open_container())
+			if(RC.is_open_container(FALSE))
```

Comment:
It's not something I really insist on to merge this but I think that for boolean arguments using named arguments helps with readability a lot. This way when I just read this line I don't know what the argument means. And even if I vaguely do remember its semantics... was it `inward` or `outward` that's true?

Reply:
all instances of inward being used now declare the argument for readability